src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMultiSignMessage.cs

```diff
@@ -422,6 +787,39 @@ public void AddSignatureForDetached(ReadOnlySpan<byte> detachedContent, CoseSign
             AddSignatureCore(detachedContent, null, signer, associatedData);
         }
 
+        /// <summary>
+        /// Adds a signature for the specified content to this message.
+        /// </summary>
+        /// <param name="detachedContent">The content to sign.</param>
+        /// <param name="signer">The signer information used to sign the content.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must also be provided during verification.</param>
+        /// <exception cref="ArgumentNullException">
+        ///   <para>
+        ///     <paramref name="detachedContent"/> is <see langword="null"/>.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     <paramref name="signer"/> is <see langword="null"/>.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="ArgumentException">
+        ///   <para>
+        ///     <paramref name="detachedContent"/> does not support reading.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     <paramref name="detachedContent"/> does not support seeking.
+        ///   </para>
```

Comment:
If you want to glue these two together into one sentence you could, since they have many words in common.

```suggestion
        ///   <para>
        ///     <paramref name="detachedContent"/> does not support reading or seeking.
        ///   </para>
```

(Yeah, it's not the first occurrence of this, but it took this long for me to decide to mention it)